etc/profile-a-l/geeqie.profile

```diff
@@ -12,6 +12,8 @@ noblacklist ${HOME}/.local/share/geeqie
 
 # Allow perl (blacklisted by disable-interpreters.inc)
 include allow-perl.inc
+# Allow lua (blacklisted by disable-interpreters.inc)
```

Comment:
Nitpick. We tend to sort things alphabetically. Please put the lua block above the perl one.

Reply:
No worries, I'm a nitpicker myself. :)